src/compiler/abstract_compiler.nit

```diff
@@ -814,6 +815,7 @@ extern void nitni_global_ref_decr( struct nitni_ref *ref );
 		v.add("signal(SIGTERM, sig_handler);")
 		v.add("signal(SIGSEGV, sig_handler);")
 		v.add("signal(SIGPIPE, sig_handler);")
+		v.add("setlocale(LC_ALL,\"\");")
```

Comment:
It sets the locale of the running program to the environment's.
We need it for language, otherwise, the locale set is by default is C's => no translation to another language.

Since it does not hurt, I left it at the beginning of the main, regardless whether a language annotation is used or not.

Since it is supposed to be called once (in theory, unless you'd like to change your locale value mid-execution, I can't really see why), I guess the main was the best place to do so, but if you have a better idea I can move it.